### PR TITLE
Adding resonatehq in deterministic simulation testing list

### PR DESCRIPTION
**Pull request by vaibhawvipul**
- added the company website
- also added the github link for their dst

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -521,6 +522,9 @@ for the Rust language ecosystem.
 
 See also [deterministic simulation](#deterministic-simulation) section.
 
+### ResonateHQ
+A dead simple programming model for [Distributed Async Await](https://www.resonatehq.io/). They use deterministic simulation testing to test their [server](https://github.com/resonatehq/resonate/tree/main/test/dst). Their testing strategy is inspired by [FoundationDB](#foundationdb) and [TigerBettle](#tigerbeetle)
```
Comment (guergabo) [inline README.md]: [TigerBeetle] != [TigerBettle]

**Comment by asatarin** [comment]
I usually don't add links to *just code*. This is less useful without larger context and / or documentation.

Is there a writeup or a talk about ideas behind the code?

**Comment by vaibhawvipul** [comment]
> I usually don't add links to _just code_. This is less useful without larger context and / or documentation.
> 
> Is there a writeup or a talk about ideas behind the code?

A write up will be available soon, this week.

**Comment by dtornow** [comment]
Hey Andrey, good timing, I just published the blog post a few hours ago :)

https://blog.resonatehq.io/deterministic-simulation-testing

**Comment by vaibhawvipul** [comment]
Hey @asatarin , I have added a link to the blog and the code. Can you please take a look once?

**Pull request closed without merging**